python/xorbits/_mars/deploy/yarn/supervisor.py

```diff
@@ -28,6 +28,12 @@ def __call__(self, *args, **kwargs):
         os.environ["MARS_CONTAINER_IP"] = self.get_container_ip()
         return super().__call__(*args, **kwargs)
 
+    def parse_args(self, parser, argv, environ=None):
+        args = super().parse_args(parser, argv, environ=environ)
+        self.config["web"]["host"] = args.endpoint.split(":", 1)[0]
```

Comment:
Does not support endpoint with schema? e.g. http://web-endpoint.com

Reply:
> Does not support endpoint with schema? e.g. http://web-endpoint.com

Here just to decide web address's host. The WebActor will handle the schema, so it doesn't matter here.

Reply:
Can ipv6 be well handled?